packages/issuer/package.json

```diff
@@ -11,6 +11,7 @@
   "dependencies": {
     "@sphereon/openid4vci-common": "workspace:../common",
     "@sphereon/ssi-types": "^0.9.0",
+    "jose": "^4.10.0",
```

Comment:
Please move jose to dev deps and copy JwtHeader params from jose to our types in the common module.

I want to stay away from using jose in non-dev imports, as certain jose functions do not work in RN environments.

I know this is the issuer, but the JwtHeader import from jose is in the common lib

Reply:
Removed!